labsync/zuul_files_sync.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash -ex
+# Copyright 2015 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+github_username=$1
+github_useremail=$2
+github_password=$3
+deploy_type=${4:-openlab}
+
+online_file=~/inotify/labkeeper/etc/zuul/${deploy_type}-main.yaml
+
+function pull_and_config_labkeeper()
+{
+    if [ ! -d ~/inotify/ ];then
+        mkdir inotify
+    fi
+    if [ ! -d ~/inotify/labkeeper/ ];then
+        echo "pull labkeeper repo"
+        git clone https://github.com/theopenlab/labkeeper ~/inotify/labkeeper
+        echo "clone labkeeper repo success!"
```

Comment:
if it have cloned the labkeeper repo previously, it will not update the repo, that may cause the local labkeeper repo out of date with the github repo, right ?

maybe you don't need to clone the repo, you can just `wget https://raw.githubusercontent.com/theopenlab/labkeeper/master/etc/nodepool/clouds-secrets.yaml` and other files to nodepool or zuul hosts every sync time. that can sure the file always latest with upstream, and do not need to depend on the whole `labkeeper` repo. Note: if you do this, you'd better to define the files' URLs as variables, in case of the directory structure of labkeeper changed.

Reply:
oh, you also need to repo to submit PR...